src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder("Boeing 737")
+                .setMass(350).setMaxMassTransported(70);
+        Plane plane1 = planeBuilder.setMaxHeight(12)
+                .setMaxSpeed(950)
+                .build();
+        Plane plane2 = planeBuilder.setMaxHeight(10)
+                .setMaxSpeed(1050)
+                .build();
+
+        System.out.println(plane1 + System.lineSeparator() + plane2);
```

Comment:
It's good to test yourself locally with main method, but let's not push it to github. Remove it in next commit